classes/Access.php

```diff
@@ -164,6 +164,24 @@ public static function findSlugByIdTab($idTab)
     }
 
     /**
+     * Find slug by Parent Tab ID
+     *
+     *
+     * @param int $idParentTab Tab ID
+     *
+     * @return string Full module slug
+     */
+    public static function findSlugByIdParentTab($idParentTab)
+    {
+        $result = Db::getInstance()->executeS('
+            SELECT `class_name`
+            FROM `'._DB_PREFIX_.'tab` t
```

Comment:
can you remove the alias if you don't use it ? if someday, we make a join.. it can be problematic!